x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -90,6 +90,7 @@ const getSavedObjectTypes = (): { [key: string]: SavedObjectsType } => ({
         fleet_server_hosts: { type: 'keyword' },
         has_seen_add_data_notice: { type: 'boolean', index: false },
         prerelease_integrations_enabled: { type: 'boolean' },
+        secret_storage_requirements_met: { type: 'boolean' },
```

Comment:
nit: I do not think we want to search on that so we can probably add `index: false`

Reply:
that's weird, getting an error in ci build about this, even though the mapping was added in this pr:

```
info Checking if mappings are compatible
--
  | │ info [o.e.c.m.MetadataCreateIndexService] [node-01] [.kibana_mappings_check] creating index, cause [api], templates [], shards [1]/[1]
  | │ info [o.e.i.m.MapperService] [node-01] [.kibana_mappings_check] reloading search analyzers
  | │ERROR There was an issue trying to apply the extracted mappings to the existing index.
  | │ERROR ResponseError: illegal_argument_exception
  | │      	Root causes:
  | │      		illegal_argument_exception: Mapper for [ingest_manager_settings.secret_storage_requirements_met] conflicts with existing mapper:
  | │      	Cannot update parameter [index] from [true] to [false]
  | │          at SniffingTransport.request (/var/lib/buildkite-agent/builds/kb-n2-2-spot-28a62e0974e60ad7/elastic/kibana-pull-request/kibana/node_modules/@elastic/transport/src/Transport.ts:535:17)
  | │          at processTicksAndRejections (node:internal/process/task_queues:95:5)
  | │          at Indices.putMapping (/var/lib/buildkite-agent/builds/kb-n2-2-spot-28a62e0974e60ad7/elastic/kibana-pull-request/kibana/node_modules/@elastic/elasticsearch/src/api/api/indices.ts:1338:12)
  | │          at checkIncompatibleMappings (check_incompatible_mappings.ts:43:18)
  | │          at run_check_mappings_update_cli.ts:66:9
  | │          at tooling_log.ts:84:18
  | │          at description (run_check_mappings_update_cli.ts:65:7)
  | │          at run.ts:70:7
  | │          at withProcRunner (with_proc_runner.ts:29:5)
  | │          at run (run.ts:69:5)


```